test/integration/testsuite_test.go

```diff
@@ -82,19 +82,14 @@ var _ = BeforeSuite(func() {
 	}
 
 	ginkgoOpts = os.Getenv("GINKGO_OPTS")
-	if err != nil {
-
+	if ginkgoOpts == "" {
 		logrus.Infof("Error: Could not read GINKGO_OPTS.")
-		logrus.Infof("%v", err)
 	}
-	Expect(err).NotTo(HaveOccurred())
+	Expect(ginkgoOpts).NotTo(BeEmpty())
 
 	pullSecretPath = os.Getenv("PULL_SECRET_PATH") // this env var should contain location of pull-secret file
-	if err != nil {
-
+	if pullSecretPath == "" {
 		logrus.Infof("Error: You need to set PULL_SECRET_PATH to find CRC useful.")
```

Comment:
Off-topic, but could be addressed:

> `"Error: You need to set PULL_SECRET_PATH to find CRC useful."`

`Error: You need to set PULL_SECRET_PATH for CRC to function properly.`

Note: found the original weirdly phrased.

Reply:
agree with you. I will change it as you suggested.